Packs/MISP/Integrations/MISPV3/MISPV3.py

```diff
@@ -561,7 +561,7 @@ def create_event_command(demisto_args: dict):
 
     if isinstance(new_event, dict) and new_event.get('errors'):
         raise DemistoException(new_event.get('errors'))
-
+    assert isinstance(new_event, MISPEvent), "The new event is not an instance of MISPEvent"
```

Comment:
@dorschw  do we allow `assert` in content?

Reply:
in rare cases, @shmuel44 let's discuss this one offline